### ISSUE TITLE
Newer Django Versions

### ISSUE THREAD

**Issue by l4rth** [evildmp/Arkestra#127]
Hi,

are you planning to integrate newer Django versions and django-cms versions?

**Comment by evildmp** [comment]
Yes, absolutely, and I hope to find some time to work on that soon.
